packages/wallet-sdk/src/vendor-js/eth-eip712-util/index.js

```diff
@@ -1,5 +1,6 @@
-const util = require('./util')
-const abi = require('./abi')
+/* eslint-disable @typescript-eslint/no-var-requires */
+const util = require('./util');
+const abi = require('./abi');
```

Comment:
seems like all these changes on this file are by linter/prettier, right?

Reply:
yeah I've ran prettier script to ensure fmt is right. I can remove it from diff if needed

Reply:
@talentlessguy gotcha. yeah, can we try minimizing the diff? i honestly don't care much about those .js files, bc we'd want to deprecate them soon anyway. 
unless it needs changes (i think i saw one of those .js files was using the old dep) let's just leave it that way.